### PR TITLE
helm: bump to version 0.12.4

### PR DESCRIPTION
**Pull request by jrobsonchase**
## What

Release ingress controller 0.10.4 and helm chart 0.12.4.

## Breaking Changes

TODO, need more changelog entries.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.10.4
+
+- Add the `--api-url` option
+  This can be used to set the endpoint for the ngrok API.
+  It can be set through via the helm `apiURL` value.
+
```
Comment (Abdiramen) [inline CHANGELOG.md:13]:
```suggestion
- Bug fix Search for backend service using the `HTTPRoute` namepace
- Set metadata for edges created by the gateway
- Add gateway to client info comment
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.10.4
+
+- Add the `--api-url` option
+  This can be used to set the endpoint for the ngrok API.
+  It can be set through via the helm `apiURL` value.
```
Comment (nikolay-ngrok) [inline CHANGELOG.md:12]:
```
### Changed

- Controller will now start without having session established. Any operations that require tunnels will return error, while it is trying to create a session. Its ready and health checks now depend on the status of this session - `ready` will not return `ok` until connection was established, and `health` check will return error if this connection had authentication issues. 
```

File: helm/ingress-controller/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.12.4
+
+- Add the `apiURL` value.
+  This sets the ngrok API endpoint used by the controller.
+  It corresponds to the `--api-url` argument to the manager binary.
```
Comment (nikolay-ngrok) [inline helm/ingress-controller/CHANGELOG.md:12]: 👀

**Review by nikolay-ngrok — CHANGES_REQUESTED** [review]

**Review by nikolay-ngrok — APPROVED** [review]

File: helm/ingress-controller/CHANGELOG.md (hunk 3 above)
Comment (El-o-heka) [reply helm/ingress-controller/CHANGELOG.md:12]: 👁️⚖️❤️

**Review by El-o-heka — APPROVED** [review]
(¥)

File: CHANGELOG.md (hunk 1 above)
Comment (El-o-heka) [reply CHANGELOG.md:29]: 😉👑🎺👁️

**Pull request merged**